locales/zh-TW/messages_po.js

```diff
@@ -1 +1 @@
cd\u7f6e\u5bc6\u78bc\uff0c\u8acb\u9ede\u64ca\u4e0b\u9762\u7684\u93c8\u63a5\uff0c\u6216\u5c07\u93c8\u63a5\u8cbc\u5230\u700f\u89bd\u5668\u3002\u7136\u5f8c\uff0c\u7cfb\u7d71\u5c07\u8981\u6c42\u60a8\u8a2d\u5b9a\u4e00\u500b\u65b0\u7684\u5bc6\u78bc\u3002","to translate":"\u7ffb\u8b6f","Tokenizer":"\u6a19\u8a18\u751f\u6210\u5668","Too many suggestions match your query. Only a few are listed":"\u592a\u591a\u7684\u5efa\u8b70\u7b26\u5408\u60a8\u7684\u67e5\u8a62\uff0c\u6240\u4ee5\u9019\u88e1\u53ea\u5217\u51fa\u5e7e\u500b\u5efa\u8b70","Tool consumer(s)":"\u5de5\u5177\u6d88\u8cbb\u8005","Translate":"\u7ffb\u8b6f","Translate to":"\u7ffb\u8b6f\u6210","Translation parameters":"\u7ffb\u8b6f\u7bc4\u570d","Translation saved":"\u7ffb\u8b6f\u5df2\u5132\u5b58","True":"\u771f","Type":"\u985e\u578b","Unable to access your account?":"\u60a8\u7121\u6cd5\u9032\u5165\u60a8\u7684\u5e33\u6236\u55ce\uff1f","Unable to change passwords in demo mode":"\u5728\u8a66\u7528\u6a21\u5f0f\u7121\u6cd5\u66f4\u6539\u5bc6\u78bc","Unable to delete the selected resource":"\u7121\u6cd5\u522a\u9664\u6240\u9078\u7684\u8cc7\u6e90","Unable to release the lock":"\u7121\u6cd5\u89e3\u9664\u9396\u5b9a","Unable to remove the property.":"\u7121\u6cd5\u79fb\u9664\u5c6c\u6027\u3002","Unable to send the password reset request":"\u7121\u6cd5\u767c\u9001\u5bc6\u78bc\u91cd\u7f6e\u8acb\u6c42","Unable to upload file":"\u7121\u6cd5\u4e0a\u50b3\u6a94\u6848","Uncheck all":"\u5168\u90e8\u53d6\u6d88\u52fe\u9078","Uncheck All":"\u5168\u90e8\u53d6\u6d88\u52fe\u9078","Undo":"\u5fa9\u539f","unknown":"\u4e0d\u660e","Unknown Error":"\u4e0d\u660e\u7684\u932f\u8aa4","Update":"\u66f4\u65b0","Upload":"\u4e0a\u50b3","Upload into:":"\u4e0a\u50b3\u5230:","Use default value":"\u4f7f\u7528\u9810\u8a2d\u503c","User":"\u4f7f\u7528\u8005","User added":"\u5df2\u65b0\u589e\u7528\u6236","User deleted successfully":"\u522a\u9664\u7528\u6236\u6210\u529f","User not found":"\u627e\u4e0d\u5230\u8a72\u4f7f\u7528\u8005","User saved":"\u7528\u6236\u5df2\u5132\u5b58","User Settings":"\u7528\u6236\u8a2d\u5b9a","Users":"\u7528\u6236","Version":"\u7248\u672c","Width":"\u5bec\u5ea6","Yes":"\u662f","You are not authorised to remove this lock":"\u60a8\u7121\u6b0a\u522a\u9664\u8a72\u9396","You cannot delete the root node":"\u60a8\u4e0d\u80fd\u522a\u9664\u6839\u7bc0\u9ede","You currently have no access to the platform":"\u60a8\u76ee\u524d\u7121\u6b0a\u9032\u5165\u5e73\u53f0","You do not have the required rights to edit this resource.":"\u60a8\u6c92\u6709\u7de8\u8f2f\u9019\u500b\u8cc7\u6e90\u7684\u6b0a\u5229\u3002","You have already taken this test.":"\u4f60\u5df2\u7d93\u4f7f\u7528\u904e\u9019\u9805\u8003\u8a66","You have finished the test!":"\u60a8\u5df2\u5b8c\u6210\u8003\u8a66","You must activate JavaScript in your browser to run this application.":"\u60a8\u5fc5\u9808\u5728\u700f\u89bd\u5668\u4e2d\u555f\u7528JavaScript\u624d\u53ef\u4ee5\u61c9\u7528\u8a72\u7a0b\u5e8f\u3002","Your mail":"\u60a8\u7684\u4fe1","Your TAO Password":"\u60a8\u7684\u5bc6\u78bc"}
```

Comment:
i think this might generate some errors

Reply:
These files get (re-)generated during compilation. And while navigating the UI after the changes I don't see any errors. I can verify this next Monday with @siwane though.